source/dbo/Stored Procedures/Data Migration/uspGenerateMigrationResultsData.sql

```diff
@@ -221,7 +221,7 @@ BEGIN TRY
 
 	UPDATE mrr
 		SET 
-			mrr.MigrationNotes = ag.[Message]
+			mrr.MigrationNotes = COALESCE(ag.[Message], ag.[Notes])
```

Comment:
I've gone for a more general approach of falling back to the Notes whenever there are no Messages. I thought that this would be most useful, in case there are other scenarios in which there are no Messages, but there are Notes.

Reply:
I'm assuming that in those cases we'd also want to see the Notes.

Reply:
Yes, this looks like a good approach